week4/MealsharingNET/Models/Meal.cs

```diff
@@ -0,0 +1,19 @@
+namespace MealsharingNET.Models;
+
+public class Meal
+{
+    public int ID { get; set; }
+    public string Title { get; set; }
+    public string Description { get; set; }
+    public string Location { get; set; }
+    public DateTime When { get; set; }
+    public int MaxReservations { get; set; }
+    public int Price { get; set; }
```

Comment:
The price can potentially be something like 9.99 so a more appropriate type would be decimal instead of int